tiledb/sm/query/query.cc

```diff
@@ -1218,10 +1218,23 @@ Status Query::init() {
 
     // Create dimension label queries and remove labels from subarray.
     if (uses_dimension_labels()) {
-      if (type_ == QueryType::WRITE && layout_ == Layout::GLOBAL_ORDER) {
+      // Check the layout is valid.
+      if (layout_ == Layout::GLOBAL_ORDER) {
```

Comment:
There isn't currently a plan to support it, but that may change.

Reply:
We use global order reads for consolidation, so that would mean we cannot consolidate? Global order should be returned in the order that the on disk data is in, so I'm not sure why we cannot just return the results in the order of the dense indexes?

Reply:
As discussed on slack, I will follow-up with a PR that enables global order.